--- SDL_mixer-1.2.7/timidity/config.h.orig	2006-05-01 09:25:52.000000000 +0200
+++ SDL_mixer-1.2.7/timidity/config.h	2006-05-20 01:17:40.326795750 +0200
@@ -176,7 +176,7 @@
 #if defined(__WIN32__) || defined(__OS2__)
 #define DEFAULT_PATH	"\\TIMIDITY"
 #else
-#define DEFAULT_PATH	"/usr/local/lib/timidity"
+#define DEFAULT_PATH	"/usr/share/timidity"
 #endif
 
 /* These affect general volume */
--- SDL_mixer-1.2.7/timidity/timidity.c.orig	2006-05-14 21:54:25.000000000 +0200
+++ SDL_mixer-1.2.7/timidity/timidity.c	2006-05-20 01:21:52.074529000 +0200
@@ -294,13 +294,11 @@
 
 int Timidity_Init(int rate, int format, int channels, int samples)
 {
-  if (read_config_file(CONFIG_FILE)<0) {
     if (read_config_file(CONFIG_FILE_ETC)<0) {
       if (read_config_file(CONFIG_FILE_ETC_TIMIDITY)<0) {
         return(-1);
       }
     }
-  }
 
   if (channels < 1 || channels == 3 || channels == 5 || channels > 6) return(-1);
 
