--- SDL_mixer-1.2.7/timidity/timidity.c.orig	2006-05-14 21:54:25.000000000 +0200
+++ SDL_mixer-1.2.7/timidity/timidity.c	2006-05-20 01:21:52.074529000 +0200
@@ -294,11 +294,9 @@
 {
   const char *env = getenv("TIMIDITY_CFG");
   if (!env || read_config_file(env)<0) {
-    if (read_config_file(CONFIG_FILE)<0) {
       if (read_config_file(CONFIG_FILE_ETC)<0) {
         return(-1);
       }
-    }
   }
 
   if (channels < 1 || channels == 3 || channels == 5 || channels > 6) return(-1);
